frontend/tests/e2e/pageObjects/studentAdmin/institute/authoritiesDetailsPage.js

```diff
@@ -4,12 +4,75 @@ const log = require('npmlog')
 class AuthoritiesDetailsPage {
   constructor() {
     this.authorityNameField = Selector('#authorityName');
+    this.editAuthorityButton = Selector('#editButton');
+    this.saveAuthorityEditButton = Selector('#saveButton');
+    this.authorityPhoneNumberInput = Selector('#phoneNumberField');
+    this.authroityPhoneNumberSpan = Selector('span');
+    this.authorityEmailInput = Selector('#emailField');
+    this.authorityMAddressLine1 = Selector('#mailAddressLine1');
+    this.authorityMAddressCity = Selector('#mailAddressCity');
+    this.authorityMAddressProv = Selector('#mailAddressProvince');
+    this.authorityMAddressCountry = Selector('#mailAddressCountry');
+    this.authorityMAddressPostal = Selector('#mailAddressPostal');
+    this.authorityEmailSpan = Selector('span');
   }
 
   async verifyAuthorityNumberAndName(name) {
     await t.expect(this.authorityNameField.withText(name).count).eql(1, {timeout: 3000});
     log.info('Authority detail name and number verified');
   }
+
+  async verifyAuthorityPhoneNumber(){
+    await t.expect(this.authroityPhoneNumberSpan.withText('101-101-1100').innerText).contains('101-101-1100');
+    log.info('Authority Phone Number Verified');
+  }
+
+  async verifyAuthorityEmailAddress(){
```

Comment:
Could we make this reusable? By accepting a text argument that is passed in to check. 

ex. 
async verifyAuthorityEmailAddress(text){
await t.expect(this.authorityEmailSpan.withText(text).innerText).contains(text);
    log.info(`Authority Email ${text} Verified`);
  }
}